sdk/core/Azure.Core/src/Pipeline/DiagnosticsOptions.cs

```diff
@@ -17,6 +18,11 @@ public DiagnosticsOptions()
 
         public bool IsTelemetryEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets value indicating if request or response content should be logged.
+        /// </summary>
+        public bool IsContentLoggingEnabled { get; set; }
```

Comment:
Should  we change the name of this to `IsLoggingContentEnabled` so they appear next to each other in intellisense?  Or should we merge the two properties into an enum?